ilan/README.md

```diff
@@ -1,37 +1,21 @@
-# Building an IBM HTTP Server v8.5.5 ILAN image from binaries
+# Building an IBM HTTP Server ILAN image from binaries
 
-An IBM HTTP Server ILAN image can be built by obtaining the following binaries:
-* IBM Installation Manager binaries from [developerWorks](http://www.ibm.com/developerworks/downloads/ws/wasdevelopers/)
+An IBM HTTP Server ILAN image can be built by running one script.
 
-  IBM Installation Manager binaries:
-  * agent.installer.linux.gtk.x86_64_1.6.2000.20130301_2248.zip
+Versions available:
+8.5.5.9
+8.5.5.10
+9.0.0.0
 
-* IBM HTTP Server, IBM WebServer Plugins and IBM WebSphere Customization Tools binaries from [developerWorks](http://www.ibm.com/developerworks/downloads/ws/wasdevelopers/) / [Fix Central](http://www-933.ibm.com/support/fixcentral/)
+If you require all available versions of IHS then simply run the script buildAll with the arguments of a IBM ID and password.
+These are required for downloading the binaries as part of the installation.
 
-  IBM HTTP Server 8.5.5 binaries:
-  * was.repo.8550.ihs.ilan_part1.zip
-  * was.repo.8550.ihs.ilan_part2.zip
+If you just want to build a specific version then you can run the build script with the version you require followed by an IBM ID and password.
 
-  IBM WebServer Plugins 8.5.5 binaries:
-  * was.repo.8550.plg.ilan_part1.zip
-  * was.repo.8550.plg.ilan_part2.zip
+You are required to download a version of IBM installation manager above version 8 and place it into the /ilan/im/ directory in this repository.
 
-  IBM WebSphere Customization Tools 8.5.5 binaries:
-  * was.repo.8550.wct.ilan_part1.zip
-  * was.repo.8550.wct.ilan_part2.zip
- 
-  Fixpack 8.5.5.9 binaries:
-  * 8.5.5-WS-WASSupplements-FP0000009-part1.zip
-  * 8.5.5-WS-WASSupplements-FP0000009-part2.zip
-  * 8.5.5-WS-WCT-FP0000009-part1.zip
-  * 8.5.5-WS-WCT-FP0000009-part2.zip
-  
-IBM HTTP Server ILAN install image is created in two steps using the following two Dockerfiles to reduce the final image size:
-
-1. [Dockerfile.prereq](https://github.com/WASdev/ci.docker.ibm-http-server/blob/master/ilan/Dockerfile.prereq)
-2. [Dockerfile.install](https://github.com/WASdev/ci.docker.ibm-http-server/blob/master/ilan/Dockerfile.install)
-
-Dockerfile.prereq perform the following actions:
+The build runs in two parts
+The first part does the following:
 
 1. Installs IBM Installation Manager
```

Comment:
These steps aren't correct any more. Personally, I wouldn't include this at all. If they want to know what it does, they can look at the source. The introductory text should, however, indicate that you're getting IHS, the plug and the customization tools.